---
title: sublime插件、快捷键
date: 2017-12-24
updated: 
tags: [sublime, 工具]
---

# 待……


# issues
1. 

> "Error: 404 Not Found
> Sorry, the requested URL 'http://127.0.0.1:51004/view/29' caused an error:

> 'buffer_id(29) is not valid (closed or unsupported file format)'

> NOTE: If you run multiple instances of Sublime Text, you may want to adjust
the server_port option in order to get this plugin work again."

Quick Fix 1: Remove Strikethrough Extension

`Sublime Text > Preferences > Package Settings > OmniMarkupPreviewer > Settings - User` 
paste the following to remove the strikeout package.
```
{
    "renderer_options-MarkdownRenderer": {
        "extensions": ["tables", "fenced_code", "codehilite"]
    }
}
```
Quick Fix 2: Fix the Strikethrough Extension (if you need it)

Find the python-markdown sublime package.

`/Packages/OmniMarkupPreviewer/OmniMarkupLib/Renderers/libs/mdx_strikeout.py`

Replace the makeExtension() method with the following:
```py
def makeExtension(*args, **kwargs):
    return StrikeoutExtension(*args, **kwargs)
Save, quit and reload Sublime Text.
```